modules/transfer-frontend/ecs.tf

```diff
@@ -18,19 +18,20 @@ data "template_file" "app" {
   template = file("modules/transfer-frontend/templates/frontend.json.tpl")
 
   vars = {
-    collector_image             = "${data.aws_ssm_parameter.mgmt_account_number.value}.dkr.ecr.eu-west-2.amazonaws.com/aws-otel-collector:${var.environment}"
-    app_image                   = "${data.aws_ssm_parameter.mgmt_account_number.value}.dkr.ecr.eu-west-2.amazonaws.com/transfer-frontend:${var.environment}"
-    app_port                    = local.app_port
-    app_environment             = var.environment
-    aws_region                  = var.region
-    client_secret_path          = var.client_secret_path
-    export_api_url              = var.export_api_url
-    backend_checks_api_url      = var.backend_checks_api_url
-    alb_ip_a                    = var.public_subnet_ranges[0]
-    alb_ip_b                    = var.public_subnet_ranges[1]
-    auth_url                    = var.auth_url
-    otel_service_name           = var.otel_service_name
-    block_draft_metadata_upload = var.block_draft_metadata_upload
+    collector_image                 = "${data.aws_ssm_parameter.mgmt_account_number.value}.dkr.ecr.eu-west-2.amazonaws.com/aws-otel-collector:${var.environment}"
+    app_image                       = "${data.aws_ssm_parameter.mgmt_account_number.value}.dkr.ecr.eu-west-2.amazonaws.com/transfer-frontend:${var.environment}"
+    app_port                        = local.app_port
+    app_environment                 = var.environment
+    aws_region                      = var.region
+    client_secret_path              = var.client_secret_path
+    export_api_url                  = var.export_api_url
+    backend_checks_api_url          = var.backend_checks_api_url
+    alb_ip_a                        = var.public_subnet_ranges[0]
+    alb_ip_b                        = var.public_subnet_ranges[1]
+    auth_url                        = var.auth_url
+    otel_service_name               = var.otel_service_name
+    block_draft_metadata_upload     = var.block_draft_metadata_upload
+    validate_draft_metadata_api_url = var.validate_draft_metadata_api_url
```

Comment:
Should we change the name to `draft_metadata_validator_api_url`? I was thinking of having a consistency between the lambda name and URL